### ISSUE TITLE
cannot create Issue  -  You must provide either the -Reporter parameter or the -Credential

### ISSUE THREAD

**Issue by 00msteiner00** [replicaJunction/PSJira#28]
HI, i tryed to create a ticket with 

```
new-JiraIssue -Project TCJ -IssueType Bug -Fields @{'Security Level'= '10401'} -Summary 'Test issue from PowerShell'  -Description 'this is the test descr. ' -Priority  '1' 
```

Return failure message:
Unable to identify issue reporter. You must provide either the -Reporter parameter or the -Credential parameter, or
the currently logged-on user must be a valid Jira user.
At C:\Program Files\WindowsPowerShell\Modules\psjira\Functions\New-JiraIssue.ps1:109 char:17
-                 throw "Unable to identify issue reporter. You must provide eithe ...
- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  - CategoryInfo          : OperationStopped: (Unable to ident...alid Jira user.:String) [], RuntimeException
  - FullyQualifiedErrorId : Unable to identify issue reporter. You must provide either the -Reporter parameter or th
    e -Credential parameter, or the currently logged-on user must be a valid Jira user.

i tryed different users -  but I'm not able to create  a ticket - to add a comment works fine

**Comment by replicaJunction** [comment]
Hello,

The issue here is that JIRA doesn't know who the reporter of the issue should be. The easiest way to fix this is to add the -Reporter parameter, like this:

`New-JiraIssue -Project TCJ -IssueType Bug -Fields @{'Security Level'= '10401'} -Summary 'Test issue from PowerShell'  -Description 'this is the test descr. ' -Priority  '1' -Reporter 'powershell-user'`

**Comment by z0ph** [comment]
Hello,

I got the same error with this : 

`New-JiraIssue -Project IPE -IssueType Incident -Priority 1 -Summary 'Test issue from PowerShell' -Description 'This is a test issue created from the PSJira module in PowerShell.' `

> Unable to identify issue reporter. You must provide either the -Reporter parameter or the -Credential parameter

I tried, like you suggested : 

`New-JiraIssue -Project IPE -IssueType Incident -Priority 1 -Summary 'Test issue from PowerShell' -Description 'This is a test issue created from the PSJira module in PowerShell.'  -Reporter = 'Lucie'

> WARNING: JIRA returned HTTP error 400 - BadRequest
Resolve-JiraError : Jira encountered an error: [reporter] - Field 'reporter' cannot be set. It is not on the appropriate 
screen, or unknown.
At C:\Program Files\WindowsPowerShell\Modules\PSJira\Internal\Invoke-JiraMethod.ps1:111 char:13
+             Resolve-JiraError $result -WriteError
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Resolve-JiraError

Do you have any idea ? Its a 'business' type Project in Jira

**Comment by replicaJunction** [comment]
Hey @z0ph - how do you normally create issues in this project through the Web interface? Do you need to specify a reporter, or does it default to something?

**Comment by z0ph** [comment]
Hey @replicaJunction, Yes, i normally create issue through web, but there is no choice of reporter, please find a screenshot below.

[https://www.dropbox.com/s/vfgb4ae4jfl6l8c/Create_Issue_-_JIRA_STVA.png?dl=0](url)

**Comment by replicaJunction** [comment]
This should be resolved in the latest version! See #64.
